client/src/pages/ClientDetailsPage.js

```diff
@@ -123,43 +127,54 @@ function ClientDetailsPage(props) {
             onClose={() => setShow(false)}
             dismissible
           >
-            Your order was successfully created!
+            {authContext.currentUser.role === UserRoles.CLIENT && ( "Your order was successfully created")}
+            {authContext.currentUser.role === UserRoles.EMPLOYEE && (`${clientInfo?.fullName}'s order was successfully created!`)}
           </Alert>
         </Row>
       ) : (
         ""
       )}
+
       {!isInitialized ? (
-        <Container className='pt-5 d-flex justify-content-center'>
-          <Spinner variant='dark' animation='border' />
+        <Container className="pt-5 d-flex justify-content-center">
+          <Spinner variant="dark" animation="border" />
         </Container>
       ) : (
+        ""
+      )}
+
+      {isInitialized && (

```

Comment:
This is useless, should be merged with the previous condition:

{!isInitialized ? spinner : content}